CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ The v1 release supports Cisco IOS-XR release version 7.7.1.
 It is planned to support 7.8.1 when this version is released.
 
 
+### v1.0.3 (2022-10-26)
+
+- Fix AppArmor bug in host-check script.
```

Comment:
Markdown consistency (sorry I'm picky on docs!). Can we be clearer about what the 'bug' is here? Users should be able to look at this file and see what's changed/fixed.
```suggestion
- Fix AppArmor bug in `host-check` script.
```